src/snowflake/snowpark/_internal/data_source/utils.py

```diff
@@ -53,6 +53,12 @@ class DBMS_TYPE(Enum):
     "sqlite3": SqliteDriver,
 }
 
+UDTF_PACKAGE_MAP = {
+    DBMS_TYPE.ORACLE_DB: ["oracledb"],
+    DBMS_TYPE.SQLITE_DB: ["sqlite3"],
+    DBMS_TYPE.SQL_SERVER_DB: ["pyodbc"],
```

Comment:
for sqlserver it would be:

```suggestion
    DBMS_TYPE.SQL_SERVER_DB: ["pyodbc", "msodbcsql"],
```